domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/casetype/CaseTypeService.java

```diff
@@ -12,6 +12,8 @@
 
     void createAll(JurisdictionEntity jurisdiction, Collection<CaseTypeEntity> caseTypes);
 
+    boolean caseTypeExists(String reference, String jurisdictionId);
```

Comment:
we have to be careful here. If we look at this method, it looks like we are checking whether a case type exists with reference `reference` and jur `jurisdictionId`
But that is not the case. This method is doing the opposite. Is checking whether any case type with `reference` exists on any jurisdiction that is not `jurisdictionId`.

Reply:
Done - excludedJurisdictionReference used here